lib/stripe/account.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Stripe.Account do
+  @moduledoc """
+  Work with Stripe account objects.
+
+  You can:
+
+  - Retrieve your own account
+  - Retrieve an account with a specified `id`
+
+  This module does not yet support managed accounts.
+
+  Stripe API reference: https://stripe.com/docs/api#account
+  """
+
+  alias Stripe.Util
+
+  @type id :: String.t
+  @type id_or_opts :: String.t | Keyword.t
+  @type t :: %__MODULE__{}
+
+  defstruct [
+    :id, :business_name, :business_primary_color, :business_url,
+    :charges_enabled, :country, :default_currency, :details_submitted,
+    :display_name, :email, :managed, :metadata, :statement_descriptor,
+    :support_email, :support_phone, :support_url, :timezone,
+    :transfers_enabled
+  ]
+
+  @singular_endpoint "account"
+  @plural_endpoint "accounts"
+
+  @doc """
+  Retrieve your own account without options.
+  """
+  @spec retrieve :: {:ok, t} | {:error, Exception.t}
+  def retrieve, do: retrieve([])
+
+  @doc """
+  Retrieve your own account with options.
+  """
+  @spec retrieve(Keyword.t) :: {:ok, t} | {:error, Exception.t}
+  def retrieve(opts), do: do_retrieve(@singular_endpoint, opts)
+
+  @doc """
+  Retrieve an account with a specified `id`.
+  """
+  @spec retrieve(binary, Keyword.t) :: {:ok, t} | {:error, Exception.t}
+  def retrieve(id, opts \\ []), do: do_retrieve(@plural_endpoint <> "/" <> id, opts)
+
+  @spec do_retrieve(String.t, Keyword.t) :: {:ok, t} | {:error, Exception.t}
+  defp do_retrieve(endpoint, opts) do
+    case Stripe.request(:get, endpoint, %{}, %{}, opts) do
+      {:ok, result} -> {:ok, to_struct(result)}
+      {:error, error} -> {:error, error}
+    end
+  end
+
+  defp to_struct(response) do
```

Comment:
You may not want this to be private, but I doubt other modules will have accounts as sub-data structures